src/services/chatWebSocket.js

```diff
@@ -124,9 +124,9 @@ export default class ChatWebSocket {
   }
 
   onOpen(callback?: Function) {
-    if (this.ws === undefined) return;
+    if (!this.ws) return;
     this.ws.onopen = () => {
-      if (typeof keepaliveTimer !== 'undefined') clearTimeout(keepaliveTimer);
+      if (keepaliveTimer) clearTimeout(keepaliveTimer);
```

Comment:
It's better to set additionally `keepaliveTimer` to null, as in JS doesn't clear the value of the timeout handler.
```javascript
if (keepaliveTimer) {
  clearTimeout(keepaliveTimer);
  keepaliveTimer = null;
}
```